src/backend/env/config.rs

```diff
@@ -346,7 +346,7 @@ pub const CONFIG: &Config = &Config {
         (12, 1),
     ],
 
-    max_funding_amount: 1_000_000, // at ratio 1:1
+    max_funding_amount: 2_000_000, // at ratio 1:1
```

Comment:
Am I right in understanding that Fund proposals are not capped, and that this cap applies just to Reward proposals?

I wonder if it could do with being renamed to `max_reward_amount`, or the cap applied to fund proposals as well (if this is the intention).

Reply:
No, it is the same cap for both.